proposals/Inflaiton Rate Change Modification Prop.md

```diff
@@ -0,0 +1,39 @@
+
+In this proposal we will be looking at adjusting the inflation rate change variable.
+
+- When the cosmos hub inflation dynamics were originally designed, the goal was for the hub to go from the minimum rate (7%) to the maximum rate (20%) in roughly one year if a shock unbond occurred. Thusly, for the variable “inflation rate change” a value of .13 was chosen. Unfortunately, in practice the variable didn’t work as intended due to the fact the change in the inflation rate for the hub is proportional to the Target Bonded / current bonded ratio.
+
+**How to select “inflation rate change” AKA Maximum Slope of the inflation curve**
+
+The purpose of changing the networks inflation rate is to protect it from unbonding shocks that can sacrifice the security of the network. When the bonded ratio gets below the goal bonded ratio (66% on the hub currently) the inflation rate & effective yield of staked atoms goes up to try and incentive new atoms to bond and secure the network. Inversely, if we regain a proper amount of staked tokens, the yield will drop and thusly decrease effective yields for all delegators. Ideally the inflation rate starts changing fast, optimizing network security over monetary hardness. To figure out what an appropriate selection for the cosmos hub would be, I created excel sheets to run through all the different scenarios to find what value made the inflation rate react in an optimal manner. **I came to the conclusion that 1 (AKA 100% per year is the maximum slope of the inflation curve) gave the most ideal characteristics, with the added benefit of simplifying the equation**. I’ll give some examples on how it would react. Since unbonding is what we are protecting against, I will look at flash unbonds while the inflation rate is at the floor due to that being the #1 time of vulnerability.
+
+#1. Cosmos hub Bonded ratio flash crashes to 60%, not bad, but 10% below the target. With our current variable of .13, it would take approx. 9.6 years to make it to the ceiling rate of 20%. On the other hand, if this variable was 1, the hub would reach it ceiling 17.2 months after the unbond shock. (assuming bonded % stays at 60% the entire time for simplicity)
+
+#2 Cosmos hub Bonded ratio flash crashes to 50%, starting to get a little scary, but only 25% below the 66% target. With our current variable of .13, it would take aprox. 4 years to make it to the ceiling rate of 20%. On the other hand, if this variable was 1, the hub would reach it ceiling 6.4 months after the unbond.
+
+#3 Cosmos hub Bonded ratio flash crashes to 35%, NOT GOOD!!!! We need to get more atoms staked ASAP! With our current variable of .13, it would take aprox 25 months to reach the ceiling rate of 20%. On the other hand, if this variable was 1, the hub would reach its ceiling 3.3 months after the unbond and the inflation rate would be increasing at a rate of 3.91% per month.

```

Comment:
These are great little descriptions but would also be great to just get a small summary table directly in here or in the spreadsheet. Ie for current (0.13) and proposed (1.0) values, show how many years it takes to get from min to max if stake drops to say 60%, 50%, 34%, or 0%. The current spreadsheet is pretty dense.